vignettes/VectorControl.Rmd

```diff
@@ -15,117 +15,124 @@ knitr::opts_chunk$set(
+    clinical_incidence_rendering_min_ages = 2*year,
+    clinical_incidence_rendering_max_ages = 10*year
   )
 )
 
-simparams <- set_equilibrium(simparams, starting_EIR)
+# Setting 3 species of mosquitoes, 25% An. arabiensis, 25% An. funestus, and 50% An. gambiae
+simparams <- set_species(
+  simparams, 
+  species = list(arab_params, fun_params, gamb_params),
+  proportions = c(0.25, 0.25, 0.5)
+)
 
-# Plotting functions
-plot_prevalence <- function(output) {
-  ggplot(output) + geom_line(
-    aes(x = timestep, y = (n_inc_severe_730_3650 / n_730_3650))) + 
-    labs(x = "timestep", y = "Severe incidence")
-}
+simparams <- set_equilibrium(simparams, starting_EIR)
 
-add_intervention_lines <- function(plot, events) {
-  plot + geom_vline(
-    data = events,
-    mapping = aes(xintercept=timestep),
-    color="blue"
-  ) + geom_text(
-    data = events,
-    mapping = aes(x = timestep, y = 0, label = name),
-    size = 4,
-    angle = 90,
-    vjust = -0.4,
-    hjust = 0
-  )
-}
+output_control <- run_simulation(sim_length, simparams)
 ```
 
-Then we can run the simulation for a variety of vector control strategies:
-
-## Bed nets
+## Simulation
 
-We can distribute bed nets once a year for two years, changing the
-characteristics of the bed nets on each distribution...
-
-```{r}
-bednetparams <- simparams
-
-bednet_events = data.frame(
-  timestep = c(1, 2) * year,
-  name=c("Bednets 1", "Bednets 2")
-)
+Having established a base set of parameters, we can now create a copy of this parameter list and update it to specify a net distribution campaign. In the example below, we distribute bednets once a year for two years, the first round to 80% of the population and the second round to 50% of the population. It is possible to change the characteristics of the bednets for each distribution timestep by modifying the values for `dn0`, `rn`, `rnm`, and `gamman`. Because there are 3 mosquito species specified in `set_species()` above and two timesteps when nets are distributed, the matrices for `dn0`, `rn`, and `rnm` have 3 columns and 2 rows.
+  
+```{r, fig.align='center', fig.height=4, fig.width=6}
+bednetstimesteps <- c(1, 2) * year # The bednets will be distributed at the end of the first and the second year. 
 
 bednetparams <- set_bednets(
-  bednetparams,
-  timesteps = bednet_events$timestep,
-  coverages = c(.8, .8),
-  retention = 5 * year,
-  dn0 = matrix(c(.533, .45), nrow=2, ncol=1),
-  rn = matrix(c(.56, .5), nrow=2, ncol=1),
-  rnm = matrix(c(.24, .24), nrow=2, ncol=1),
-  gamman = rep(2.64 * 365, 2)
+  simparams,
+  timesteps = bednetstimesteps,
+  coverages = c(.8, .5),  # The first round is distributed to 80% of the population and the second round to 50%.
+  retention = 5 * year, # Nets are kept on average 5 years
+  dn0 = matrix(c(.533, .45), nrow=2, ncol=3), # Matrix of death probabilities for each mosquito species over time
```

Comment:
I changed the values so that they were the same for each timestep as I was not sure what the values corresponded to. The DOI in the documentation pointed me to the same paper that the IRS model was based on, which I'm not sure is correct since it doesn't seem to talk about nets. 
"The structure for the bed net model is documented in the S.I. of 10.1038/s41467-018-07357-w"